website/source/guides/configuration/index.html.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: "guides"
+page_title: "Vault Configuration - Guides"
+sidebar_current: "guides-configuration"
+description: |-
+  Once a Vault instance has been installed, the next step is to configure auth
+  backend, secret backend, and manage keys. Vault configuration guides addresses
+  key concepts in configuring your Vault servers.   
+---
+
+# Vault Configuration
+
+This guide walks you through Vault configuration topics. Some guides address
+fundamental tasks to get the server setup, and some guides introduce more
```

Comment:
It's less about getting the server setup, and more about getting your Vault application configured